math/genvector/inc/Math/GenVector/LorentzVector.h

```diff
@@ -731,6 +731,69 @@ ROOT provides specialisations and aliases to them of the ROOT::Math::LorentzVect
        return tmp;
     }
 
+  /**
+     pair (p+ p-) acoplanarity `alpha = 1 - |phi+ - phi-|/pi`.
+     \param pp p+, mathcore::LorentzVector based on any coordinate system
+     \param pm p-, mathcore::LorentzVector based on any coordinate system
+     \return a scalar
+     \see http://doi.org/10.1103/PhysRevLett.121.212301
+   */
+    template< class CoordSystem >
+    typename LorentzVector<CoordSystem>::Scalar Acoplanarity(LorentzVector<CoordSystem> pp, LorentzVector<CoordSystem> pm)
+    {
+       auto dphi = pp.Phi() - pm.Phi();
+       // convert dphi angle to the interval (-PI,PI]
+       if (dphi > TMath::Pi() || dphi <= -TMath::Pi()) {
+          if (dphi > 0) {
+             int n = static_cast<int>(dphi / TMath::TwoPi() + 0.5);
+             dphi -= TMath::TwoPi() * n;
+          } else {
+             int n = static_cast<int>(0.5 - dphi / TMath::TwoPi());
+             dphi += TMath::TwoPi() * n;
+          }
+       }
```

Comment:
Maybe with `fmod`?
```suggestion
    if (dphi <= -TMath::Pi()) dphi =  TMath::Pi() + std::fmod(dphi, TMath::Pi());
    if (dphi >   TMath::Pi()) dphi = -TMath::Pi() + std::fmod(dphi, TMath::Pi());
```
`fmod` preserves the sign, so in the first case, we move left from the upper bound, in the second case right from the lower bound.

Reply:
Aren't two if-checks slightly better than fmod from the performance point of view?

This was a copy of
Math/GenVector/LorentzVector.h:DeltaR
as well as
Math/GenVector/VectorUtil.h
as well as
math/physics/src/TVector2.cxx

Should the three (now four) implementations be all changed to fmod ? Or only this one?